tests/ut_seattlelibv2_repypp_include_non-existingfiles.py

```diff
@@ -0,0 +1,34 @@
+#pragma out Error opening source file 'nonexit.py'
```

Comment:
Small typo: We want to refer to a non-existing file rather than a non-exiting (note "s").